src/commands/synthetics/__tests__/cli.test.ts

```diff
@@ -124,6 +130,9 @@ describe('run-test', () => {
         tunnel: true,
         variableStrings: ['key=value'],
       }
+      const defaultTestOverrides: UserConfigOverride = {
```

Comment:
+1 for the name :grin:

Reply:
Yeah decided to make it easier for myself for later 😂